internal/app/machined/pkg/controllers/k8s/templates.go

```diff
@@ -203,390 +56,34 @@ subjects:
 // these env vars, such as kube-proxy, which sets up the API server endpoint
 // (chicken and egg), and the checkpointer, which needs to run as a static pod
 // even if the API server isn't available.
-var kubeConfigInClusterTemplate = []byte(`apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: kubeconfig-in-cluster
-  namespace: kube-system
-data:
-  kubeconfig: |
-    apiVersion: v1
-    clusters:
-    - name: local
-      cluster:
-        server: {{ .Server }}
-        certificate-authority: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
-    users:
-    - name: service-account
-      user:
-        # Use service account token
-        tokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
-    contexts:
-    - context:
-        cluster: local
-        user: service-account
-`)
-
-var coreDNSTemplate = []byte(`apiVersion: v1
-kind: ServiceAccount
-metadata:
-  name: coredns
-  namespace: kube-system
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRoleBinding
-metadata:
-  name: system:coredns
-  labels:
-    kubernetes.io/bootstrapping: rbac-defaults
-  annotations:
-    rbac.authorization.kubernetes.io/autoupdate: "true"
-roleRef:
-  apiGroup: rbac.authorization.k8s.io
-  kind: ClusterRole
-  name: system:coredns
-subjects:
-  - kind: ServiceAccount
-    name: coredns
-    namespace: kube-system
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: ClusterRole
-metadata:
-  name: system:coredns
-  labels:
-    kubernetes.io/bootstrapping: rbac-defaults
-rules:
-  - apiGroups: [""]
-    resources:
-      - endpoints
-      - services
-      - pods
-      - namespaces
-    verbs:
-      - list
-      - watch
-  - apiGroups: ["discovery.k8s.io"]
-    resources:
-      - endpointslices
-    verbs:
-      - list
-      - watch
----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: coredns
-  namespace: kube-system
-data:
-  Corefile: |
-    .:53 {
-        errors
-        health {
-            lameduck 5s
-        }
-        ready
-        log . {
-            class error
-        }
-        prometheus :9153
+//
+//go:embed templates/kube-config-in-cluster.yaml
```

Comment:
Maybe to keep up with the naming scheme?

```suggestion
//go:embed templates/kube-config-in-cluster-template.yaml
```